lib/camerata/parameters.rb

```diff
@@ -1,14 +1,24 @@
 # frozen_string_literal: true
 module Camerata
   class Parameters
-    # key can be any list of names, comma seperated
+    # key can be any list of up to ten names, separated by spaces
     def self.get(key)
       raise 'please set your AWS_PROFILE and AWS_DEFAULT_REGION' unless ENV['AWS_DEFAULT_REGION'] && ENV['AWS_PROFILE']
       key = "\"#{key}\"" unless key.match?('"')
       result = call_aws_ssm(key)
       JSON.parse(result) if result && !result.empty?
     end
 
+    def self.get_list(parameter_list)
+      raise 'please set your AWS_PROFILE and AWS_DEFAULT_REGION' unless ENV['AWS_DEFAULT_REGION'] && ENV['AWS_PROFILE']
+      result_list = []
+      parameter_list.each_slice(10) do |slice|
+        result = call_aws_ssm(slice.join(" "))
+        result_list << JSON.parse(result) if result && !result.empty?
+      end
+      result_list
+    end
+
     def self.call_aws_ssm(key)
       `aws ssm get-parameters --names #{key} --with-decryption`
```

Comment:
could this be the issue? since this is executed in a shell, you may need to surround #{key} with single quotes.